### ISSUE TITLE
Resources README is about epinowcast

### ISSUE THREAD

**Issue by athowes** [epinowcast/epidist#10]
The [Resources section](https://github.com/epinowcast/epidist/tree/main?tab=readme-ov-file#resources) of the README file for this package is about `epinowcast` rather than `epidist`. I would expect that this section should be updated to be about `epidist`, or if information about `epinowcast` is provided it be more clear why it is relevant.

**Comment by seabbs** [comment]
Yes this (and other similar things are due to the infrastructure only being partially ported over from `epinowcast`. I think a sensible step here is to scope out what other things like this there are and assemble into a meta-issue with child issues for each piece of work.
